packages/components/src/Account/Auth/AuthStyledComponents.tsx

```diff
@@ -162,3 +165,27 @@ export const AuthEmailVerify = ({
     </>
   );
 };
+
+// Staging: jorge+011012@civil.co exists. jorge@civil.co does not.
+//
+export const AuthErrorMessage = styled.div`
+  border: 1px solid rgba(242, 82, 74, 0.56);
+  border-radius: 4px;
+  background-color: #fff7f8;
+
+  background-position: 10px center;
+  background-image: url(${iconError});
+  background-size: 30px;
+  background-repeat: no-repeat;
+
+  color: #555555;
+  font-family: "Libre Franklin";
```

Comment:
Can you use the font style constant `font-family: ${fonts.SANS_SERIF};`

Reply:
Ooff, good catch.